flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/translators/SinkTransformationTranslator.java

```diff
@@ -173,61 +185,76 @@ private void expand() {
             }
         }
 
-        private <CommT> void addCommittingTopology(Sink<T> sink, DataStream<T> inputStream) {
```

Comment:
Can we please refert the generic parameter name change? This would avoid some unnecessary changes in the commit